### ISSUE TITLE
This library no longer works in a browser

### ISSUE THREAD

**Issue by kaiyoma** [silverwind/port-numbers#13]
The newest version of this library depends on `node:fs`, which does not work in a browser. After upgrading, my Vite build fails:

```
19:50:14  rush build (1.48 seconds)
19:50:16  vite v4.3.8 building for production...
19:50:16  transforming...
19:51:12  [plugin:vite:resolve] Module "node:fs" has been externalized for browser compatibility, imported by "/home/jenkins/workspace/Geiger/geiger-review/node_modules/.pnpm/port-numbers@7.1.0/node_modules/port-numbers/index.js". See http://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.
...
19:54:06  ✓ 20229 modules transformed.
19:54:06  ✓ built in 3m 50s
19:54:06  "readFileSync" is not exported by "__vite-browser-external", imported by "node_modules/.pnpm/port-numbers@7.1.0/node_modules/port-numbers/index.js".
19:54:06  file: /home/jenkins/workspace/Geiger/geiger-review/node_modules/.pnpm/port-numbers@7.1.0/node_modules/port-numbers/index.js:1:8
19:54:06  1: import {readFileSync} from "node:fs";
```

Node APIs should not be used in dependencies meant for the browser.

**Comment by silverwind** [comment]
Likely will bundle the json files into the exported file with esbuild.

**Comment by silverwind** [comment]
Fixed in 8.0.0.
